docs/spec/operations.md

```diff
@@ -23,28 +23,28 @@ Sidetree Methods MAY rely on the DID Document data model when verifying protocol
 
 Use the following process to generate a Sidetree-based DID:
 
-1. Generate a recovery key pair via the [`KEY_ALGORITHM`](#key-algorithm), retaining the [_Initial Recovery Public Key_](#initial-recovery-key){id="initial-recovery-key"} for use in generating the _Recovery Commitment_, and the private key for use in the next [Recovery](#recovery) operation.
-2. Create a _Recovery Commitment_ using the [Hashing Process](#hashing-process) to generate a hash value from the [JCS canonicalized](https://tools.ietf.org/html/draft-rundgren-json-canonicalization-scheme-17) [IETF RFC 7517](https://tools.ietf.org/html/rfc7517) JWK representation of the [_Initial Recovery Public Key_](#initial-recovery-key), and retain the hash value for inclusion in an [Anchor File](#anchor-file) (if publication of the DID is desired).
-3. Generate and retain a [`COMMITMENT_VALUE`](#commitment-value), in adherence with the [Commitment Value Generation](#commitment-value-generation) directives, for use in the next Update operation, herein referred to as _Update Reveal Value_.
-4. Create an _Update Commitment_ using the [Hashing Process](#hashing-process) to generate a hash of the _Update Reveal Value_, in adherence with the [Commitment Value Generation](#commitment-value-generation) directives, and retain the resulting hash value for inclusion in an [Anchor File](#anchor-file) (if publication of the DID is desired).
+1. Generate a [recovery key pair](#recovery-key-pair) using the defined [`KEY_ALGORITHM`](#key-algorithm), where the public key of this pair is used for generating the [recovery commitment](#recovery-commitment), and the private key for use in the next [recovery](#recovery) operation.
+2. Generate a [recovery commitment](#recovery-commitment) using the defined [recovery commitment scheme](#recovery-commitment-scheme) and public key of the generated [recovery key pair](#recovery-key-pair).
+3. Generate an [operation commit value](#operation-commit-value), let the result be known as the _Update Operation Commit Value_.
+4. Generate an [operation commitment](#operation-commitment) using the _Update Operation Commit Value_ as the [operation commit value](#operation-commit-value), let the result be known as the _Update Operation Commitment_.
 5. Generate an encoded representation of the following object using the implementation's [`DATA_ENCODING_SCHEME`](#data-encoding-scheme), herein referred to as the [_Create Operation Delta Object_](#create-delta-object){ id="create-delta-object" }:
     ```json
     {
       "patches": [ PATCH_1, PATCH_2, ... ],
-      "update_commitment": HASH_OF_UPDATE_COMMITMENT_VALUE
+      "update_commitment": Update Operation Commitment
```

Comment:
I'd prefer to see a real world example here,

Reply:
Yeah I agree in general I think real world examples here is a good idea

Reply:
@csuwildcat @thehenrytsai can you please pull real world values for this and share them on https://github.com/decentralized-identity/sidetree/issues/737

Reply:
don't bother, we can fix this stuff later.